Removed the layout item referencing Account.UpsellOpportunity__c; all unrelated metadata and ordering were preserved.

--- before/backup/layouts/Account-Account Layout.layout-meta.xml
+++ after/working/layouts/Account-Account Layout.layout-meta.xml
@@ -126,10 +126,6 @@
             <layoutItems>
                 <behavior>Edit</behavior>
                 <field>SLASerialNumber__c</field>
-            </layoutItems>
-            <layoutItems>
-                <behavior>Edit</behavior>
-                <field>UpsellOpportunity__c</field>
             </layoutItems>
         </layoutColumns>
         <style>TwoColumnsLeftToRight</style>
@@ -430,4 +426,4 @@
         <sizeY>0</sizeY>
         <summaryLayoutStyle>Default</summaryLayoutStyle>
     </summaryLayout>
-</Layout>
+</Layout>
\ No newline at end of file
